### PR TITLE
docs(blog): add blog

### PR DESCRIPTION
**Pull request by lfbergee**
affects: @fremtind/portal

## 📥 Proposed changes

Sometimes it might be easier to provide information and documentation in a more informal way, than adding it to the main section of the portal. A blog can be a great place to share ideas and experiences. I've added a small tidbit of my experience using framer motion to add animations to the portal, and a post on how to create a post. This is very much a base setup, we'll run in to issues with the sidebar and main menu when the number of posts grow, however, that should be pretty easy solvable when we get there. We can also expand with a custom layout, more metadata and other integrations going forward. It might be a bit intimidating to write blogposts with markdown, instead of a traditional CMS, but our designers has already done this with the documentation, so I have full confidence that they'll master this as well.

## ☑️ Submission checklist

-   [x] I have read the [CONTRIBUTING](https://github.com/fremtind/jokul/blob/master/CONTRIBUTING.md) doc
-   [x] `yarn build` works locally with my changes
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: portal/src/components/Sidebar/SidebarMenuItem.tsx [hunk 1]
```diff
@@ -20,9 +19,18 @@ export function SidebarMenuItem({ idx, path, title }: SidebarMenuItemProps) {
...
                 {title}
             </Link>
+            {author && (
+                <p
+                    className="jkl-micro jkl-spacing--bottom-1"
+                    data-testid={`sidebar-author-${title.replace(/ /g, "-")}`}
+                >
+                    {author} - {publishDate}
+                </p>
+            )}
```
[the remark this answers is not in the document]
Comment (lfbergee) [reply portal/src/components/Sidebar/SidebarMenuItem.tsx:33]: Det er ulempen med å flytte layout ut av dataloopen. Dermed har ikke layout tilgang på den daten, men kan jo gjøre det med en komponent i hver enkelt blog.mdx da.

File: portal/src/components/Header/Header.tsx [hunk 2]
```diff
@@ -52,6 +52,9 @@ export const Header = ({ className }: { className?: string }) => {
                     <li className="jkl-portal-header__navigation-item">
                         <FullScreenMenu title="Kom i gang" items={getStartedDocPages} activePath={PageType.KOMIGANG} />
                     </li>
+                    <li className="jkl-portal-header__navigation-item">
+                        <FullScreenMenu filterable title="Blog" items={blogPages} activePath={PageType.BLOG} />
```
[the remark this answers is not in the document]
Comment (lfbergee) [reply portal/src/components/Header/Header.tsx:56]:
```suggestion
                        <FullScreenMenu filterable title="Blogg" items={blogPages} activePath={PageType.BLOG} />
```

File: portal/src/components/Header/useNavigationLinks.tsx [hunk 3]
```diff
@@ -1,21 +1,23 @@
...
-                title: string;
-                order?: string;
-            };
+            frontmatter: Frontmatter;
         };
     };
 }
 
-export interface DocumentationPageInfo {
+export interface DocumentationPageInfo extends Frontmatter {
```
[the remark this answers is not in the document]
Comment (lfbergee) [reply portal/src/components/Header/useNavigationLinks.tsx:19]: Foreslår det gjøres i en annen pr

File: portal/integration/blog.spec.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+context("Accordion", () => {
+    beforeEach(() => {
+        cy.visit("/");
+    });
+
+    it("Accordion should work", () => {
```
[the remark this answers is not in the document]
Comment (lfbergee) [reply portal/integration/blog.spec.js:6]:
```suggestion
context("Accordion", () => {
    beforeEach(() => {
        cy.visit("/");
    });

    it("Blog should work", () => {
```

File: portal/integration/blog.spec.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+context("Accordion", () => {
```
Comment (lfbergee) [inline portal/integration/blog.spec.js:1]:
```suggestion
context("Blog", () => {
```

File: portal/src/components/Sidebar/SidebarMenuItem.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (piofinn) [reply portal/src/components/Sidebar/SidebarMenuItem.tsx:33]: Disse sidene bruker `ComponentDocLayout`-komponenten som wrapper (i tillegg til `Layout`), så man kan få inn informasjonen der på samme måte som GitHub-linkene 😊 Den komponenten burde sikkert også omdøpes i så fall.

File: portal/integration/blog.spec.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
+context("Blog", () => {
+    beforeEach(() => {
+        cy.visit("/");
+    });
+
+    it("Blog should work", () => {
+        cy.getByTestid("header-Blog").click();
```
Comment (piofinn) [inline portal/integration/blog.spec.js:7]:
```suggestion
        cy.getByTestid("header-Blogg").click();
```
Feiler nå som tittelen er endret

**Pull request merged**